llarp/router/router.cpp

```diff
@@ -870,6 +874,58 @@ namespace llarp
     m_LastStatsReport = now;
   }
 
+  std::string
+  Router::status_line()
+  {
+    std::string status;
+    auto out = std::back_inserter(status);
+    fmt::format_to(out, "v{}", llarp::VERSION_STR);
```

Comment:
This appears to be the only place that uses `VERSION_STR` in the entire code base; should we just drop it entirely and use fmt here to join the numeric array like this?
```c++
fmt::format_to(out, "v{}", fmt::join(llarp::VERSION, "."));
```